src/adapter/input/post_router.py

```
@@ -47,6 +47,23 @@ def delete_post(post_id:str, authentication_code = Header(None, convert_undersco
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Error in request process: {str(e)}") from e
+        
+@router.patch(path='/{post_id}',
+            status_code=status.HTTP_200_OK,
+            responses={
+                403: {"description": "Authentication Code Error"},
+                404: {"description": "Post Not Found"},
+                500: {"description": "Internal Server Error"}
+            })
+def update_post(post_id:str, post_update_request: PostUpdateRequest, service: PostApplicationService = Depends(get_post_application_service)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)